GraphQL.PreProcessingExtensions/Paging/CursorPaging/CursorPageSlice.cs

```diff
@@ -15,7 +15,7 @@ public class CursorPageSlice<TEntity> : ICursorPageSlice<TEntity>
     {
         public CursorPageSlice(IEnumerable<ICursorResult<TEntity>> results, int? totalCount, bool hasPreviousPage, bool hasNextPage)
         {
-            this.CursorResults = (results ?? throw new ArgumentException(nameof(results))).ToList();
+            this.CursorResults = results ?? Enumerable.Empty<ICursorResult<TEntity>>();
```

Comment:
This change will make this inconsistent with `OffsetPageResults` whereby both constructors validate that a results enumerable must be specified.  Though the `.ToList()` here does look unnecessary... I'll validate locally and remove it.

But the following changes to the Custom Extensions look to be good fixes so that they are null safe 👍

```suggestion
            this.CursorResults = results ?? throw new ArgumentException(nameof(results));
```

Reply:
Sure thing - let's have the null check 👍